### DISCUSSION TITLE
Some instructions on building the samples might be helpful

### DISCUSSION THREAD

**Discussion by cku239** [DiligentGraphics/DiligentSamples#41] [General]
So I tried to run samples,hmm no instructions. Not being familiar with CMake I really didn't want to have to learn this just top get a basic sample working, but what the, I thought give it a go.

So following CMake instructions on building project with CMakeLists.tx I tried again.

error no supported backends:661.  Looks like VULKAN_SUPPORTED and GL_SUPPORTED not being set. Where do they get set. Am I missing some basic step, I dont know because there is literally zero instructions on getting your tutorials to work.

Yeah I could probably work it out, but I would rather spend time on actually getting actual work done using a library where I don't have to waste time just getting basic examples to work due to a total lack of any instructions.

Maybe I'm just not the intended audience for this.

**Comment by TheMostDiligent** [comment]
[Build instructions](https://github.com/DiligentGraphics/DiligentEngine#build-and-run-instructions) give very detailed descriptions of the steps required to build the engine on every platform. As the project is supplied in source code, you need have a basic understand of how to set up the build environment on your platform.

There are [some additional details](https://github.com/DiligentGraphics/DiligentSamples#build-and-run-instructions) for samples and tutorials.
